<launch>
	<!-- Launches the tum_ardrone StateEstimator -->
	<node name="drone_stateestimation" pkg="tum_ardrone" type="drone_stateestimation" output="screen" clear_params="true">
	    <param name="publishFreq" value="30" /> <!-- Publish Frequency --> 
	</node>
	
	<!-- Configure Dynamically Reconfigurable Parameters -->
	<node name="dynparam" pkg="dynamic_reconfigure" type="dynparam" args="set /drone_stateestimation UsePTAM false">
	</node>
</launch>
